src/m365/planner/commands/task/task-list.ts

```diff
@@ -14,20 +26,193 @@ class PlannerTaskListCommand extends GraphItemsListCommand<Task> {
   }
 
   public get description(): string {
-    return 'Lists Planner tasks for the currently logged in user';
+    return 'Lists the planner tasks in a bucket, plan, or tasks for the currently logged in user';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.bucketId = typeof args.options.bucketId !== 'undefined';
+    telemetryProps.bucketName = typeof args.options.bucketName !== 'undefined';
+    telemetryProps.planId = typeof args.options.planId !== 'undefined';
+    telemetryProps.planName = typeof args.options.planName !== 'undefined';
+    telemetryProps.ownerGroupId = typeof args.options.ownerGroupId !== 'undefined';
+    telemetryProps.ownerGroupName = typeof args.options.ownerGroupName !== 'undefined';
+    return telemetryProps;
   }
 
   public defaultProperties(): string[] | undefined {
-    return ['id', 'title', 'startDateTime', 'dueDateTime', 'completedDateTime'];
+    return ['id', 'title', 'startDateTime', 'dueDateTime', 'completedDateTime', 'planId', 'bucketId'];
```

Comment:
This is too much to fit in the terminal window and makes the output unreadable. Let's stick with the original set for now.